### ISSUE TITLE
GGMax Enhancement - Increase maximum storyboard screens  from the current hard limit of 50 as some devs projects now cannot continue

### ISSUE THREAD

**Issue by mav3r1ck1981** [TheGameCreators/GameGuruRepo#5865]
This has been brought up previously, but it has one become an issue with several developers and their projects, including myself.

There appears to be a hard limit on the maximum screens that can be used in the storyboard, that limit is 50. Myself and @C4Ever08 are now getting "max limit" notifications and cannot add new screens of any type. For me at least, i need a new loading screen for each level, it would save me 15 screens if i could link each new level to the same loading screen, meaning that the loading screen is used multiple times.

15 level screens, plus all the hud screens and menu screens, plus loading screens and you hit the max of 50 storyboard screens pretty easily. While you can still make new levels, you cannot get them to function on the storyboard and complete the project.

**Comment by Kitakazi** [comment]
Ye I hit 50 screens a while ago too. I also use HUD screens for in-game monitor screens that the player can interact with. I could easily have 100 of just those alone.

**Comment by mav3r1ck1981** [comment]
@Kitakazi Im doing exactly the same, but with scrying pools that perform a similar function. Works quite nicely! But i cant go any further now

**Comment by Necrym59** [comment]
Why is so many screens wasted on loading screens??

**Comment by mav3r1ck1981** [comment]
If we were able to add in the rotating hints from a library on a loading screen, as has been discussed in the past, this would add the variety that multiple loading screens can currently offer, with the convenience of only using a single one.

**Comment by LeeBamberTGC** [comment]
@plemsoft Yes we need to make sure its forward and backward compatible if we update the storyboard structure, and I would also recommend we remove the hard coded limits, replacing them with an expandable list so we don't have to increase this limit again in the future.

**Comment by Necrym59** [comment]
The number of User globals too should have no limitations as well.

**Comment by mav3r1ck198-1** [comment]
@Kasseyus Preben lifted this limit back in January so this can be closed. For some reason, Github wont let me close my own posts or id do it myself :(
